### ISSUE TITLE
trailing_comma_tuple false negative with 'yield'

### ISSUE THREAD

**Issue by mthuurne** [PyCQA/pylint#2363]
### Steps to reproduce

Run pylint on this code:
```python
def gen():
    yield 'hello',
```

### Current behavior

No `trailing_comma_tuple` message is issued.

### Expected behavior

A `trailing_comma_tuple` message would be issued.

If you change `yield` to `return`, the message is issued, so for some reason `yield` is treated differently.

### pylint --version output

```python
pylint 2.0.0
astroid 2.0.0.dev4
Python 3.4.6 (default, Mar 22 2017, 12:26:13) [GCC]
```
